kcauto-kai.sikuli/repair.sikuli/repair.py

```diff
@@ -254,11 +254,15 @@ def _update_combat_next_sortie_time(self):
         """Method to update the Combat module's next sortie timer based on the
         shortest timer in the internal repair timers list.
         """
-        self.repair_timers.sort()
-        shortest_timer = self.repair_timers[0]
+        if len(self.repair_timers) > 0:
+            if len(self.repair_timers) > 1:
+                self.repair_timers.sort()
+            shortest_timer = self.repair_timers[0]
+        else:
```

Comment:
This conditional branch should be removed. Otherwise it will overwrite any other delay set by the combat module such as full port or `COMBAT_BUFFER_MINUTES`. In general, the repair module should defer to the combat module in regards to shortening the combat timer, and only delay (see PR review note for additional issues with the logic I implemented previously).